### ISSUE TITLE
FR: Clients should not allow you to select exit nodes that are blocked by ACL packet filters

### ISSUE THREAD

**Issue by samlinville** [tailscale/tailscale#8630]
### What are you trying to do?

It took me quite a long time to debug why my exit nodes were broken, and it turned out to be that I was missing the `autogroup:internet` in my ACLs. Because I technically had access to the node itself, it showed up in my list of nodes and available exit nodes, but would break my internet connection every time I selected the exit node.

### How should we solve this?

We should not allow users to select exit nodes that won't actually work for them. Ideally, we could surface an error to help people understand why they can't select those exit nodes so that they know where to start debugging (in their ACL file)

### What is the impact of not solving this?

If exit nodes break, it can be really confusing as to why that happened.

**Comment by bradfitz** [comment]
I don't think we can or should hide exit nodes that don't have `autogroup:internet`, as that would break people using exit nodes to permit access to a subset of the internet.

We could push down a health check warning to the node. There's an open bug somewhere to surface those in GUI clients, but they'd at least show up in `tailscale status`.

Also, we could show warnings in the admin console in various places.

**Comment by DentonGentry** [comment]
Duplicate of https://github.com/tailscale/tailscale/issues/1567
